plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -65,6 +72,11 @@ public IcebergTableProperties(IcebergConfig icebergConfig)
                         "File system location URI for the table",
                         null,
                         false))
+                .add(integerProperty(
```

Comment:
Added support for users to directly create a v2 table using property `format_version`

Reply:
Does this mean that I can define the iceberg table spec in the **with** clause by specifying **format_version**?

Reply:
yes